test/utils/redux.js

```diff
@@ -12,10 +13,13 @@ function restoreAppStore() {
 }
 
 export function createMockedStore(sinon, mockedState = {}) {
-    const store = createStore(() => mockedState);
+    const middlewares = [thunkMiddleware];
+    const mockStore = configureStore(middlewares);
+    const store = mockStore(mockedState);
 
     sinon.spy(store, 'dispatch');
     store.restore = restoreAppStore;
+    store.subscribe = () => Function();
```

Comment:
@liyefei737 it doesn't subscribe to an empty function, it replaces `subscribe` with an empty one. I'd actually do `store.subscribe = sinon.stub()` instead in case we need to test it somehow.